### ISSUE TITLE
Cuckoo is stuck at "Checking for updates ..." for a long time now

### ISSUE THREAD

**Issue by salijona** [cuckoosandbox/cuckoo#3163]
I am trying lately to setup cuckoo sandbox and having some issues when starting it. 
After running the command cuckoo the program is stuck for more than one hour in "Checking for updates ..." even after I set 
version_check = no and ignore_vulnerabilities = yes in the file cuckoo.conf

What would be a normal time to check for updates ?

<img width="672" alt="updates" src="https://user-images.githubusercontent.com/30389685/104902125-9970f400-597e-11eb-962d-d6ed15d69665.PNG">

**Comment by ADumpling** [comment]
Maybe you can check if there is something wrong with your configuration files.
I had the same problem last night and wasted the whole night waiting for the so-called update.
When I went to other people's tutorials about how to set up cuckoo, I found that I wrote a wrong path of the VBoxManage utility in the file virtualbox.conf.(I use VBox to run the virtual machine)
It should be 'C:\Program Files\Oracle\VirtualBox\\**VBoxManage.exe**'  while I used 'C:\Program Files\Oracle\VirtualBox\\**VirtualBox.exe**' instead.
After fixing this bug, I found cuckoo can run successfully.

**Comment by salijona** [comment]
I gave up configuring it my self after so many failed attempts. I used BoomBox to deploy a malware analysis environment with Cuckoo and Windows 10 as a guest VM. It took only 30 minutes.

However, after I read your comment I went to check my virtualbox.conf file and I had exactly the same mistake as you. I feel sorry that I did not realize such a simple thing before.  Your comment is very valuable for all people like me who did not carefully examine their config files.
